### DISCUSSION TITLE
Langkah Membuat Flexdashboard

### DISCUSSION THREAD

**Discussion by mosesnugraha** [teamalgoritma/community#564] [Q&A] [answered]
1.  Pilih Menu File \> New File \> R Markdown
    2.  Pada bagian "From Template", pilih "Flex Dashboard"
    3.  Klik tombol "OK"
    4.  Simpan file Rmd dan beri nama file (bisa dengan shortcut CTRL + S

saya mau bertanya setelah langkah ini dikerjakan, lalu kan muncul Rscript baru atau rmd baru gitu kan, nah di script baru ini ditulis lagi kań code2nya buat si plot2 di flexdasboard ini??

**Comment by ranydwi** [comment, marked as answer]
Dear Pak @mosesnugraha 

Untuk pertanyaan yang masuk, jawabannya betul Pak Moses. Silahkan untuk chunk yang kosong diisikan visualisasi yang ingin Bapak buat

Regards.
